autoload/neomake/makers/ft/text.vim

```diff
@@ -27,3 +34,51 @@ function! neomake#makers#ft#text#writegood() abort
                 \ 'postprocess': function('neomake#makers#ft#text#PostprocessWritegood'),
                 \ }
 endfunction
+
+let s:languagetool_fallback_language = 'auto'
+" See http://wiki.languagetool.org/public-http-api for a public instance. Use:
+"   :let g:neomake_text_languagetool_server = 'https://languagetool.org/api'
+let s:languagetool_fallback_server = 'http://localhost:8081'
```

Comment:
Those could go into the function directly.

Reply:
Done!